<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndSignUpWidget</class>
 <widget class="QWidget" name="FindAndSignUpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 189, 137, 236), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLineEdit" name="MailLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 1px solid black;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="KeyLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendKeyPushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterKeyPushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>240</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password1LineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password2LineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterPushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>240</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
